README.md

```diff
@@ -459,44 +482,83 @@ them to the `'authproc'` array in your `metadata/saml20-idp-hosted.php` file.
 
 Example (for `metadata/saml20-idp-hosted.php`):
 
-    use Sil\PhpEnv\Env;
-    use Sil\Psr3Adapters\Psr3SamlLogger;
+```php
+use Sil\PhpEnv\Env;
```

Comment:
Perhaps it is github thinking it is supposed to be an ANSI escape code, but highlighting it, because it isn't?